marlowe-dashboard-client/src/Contract/Types.purs

```diff
@@ -20,11 +20,15 @@ data PreviousStepState
   = TransactionStep TransactionInput
   | TimeoutStep Slot
 
+-- FIXME: Is contract id just a string? Should we wrap-it in a newtype?
+type ContractId
+  = String
+
```

Comment:
yeah, we want `ContractInstanceId`, which is a newtype arround `UUID` - but we can sort this out later